MatchingGame.js

```diff
@@ -27,17 +25,17 @@ const getData = () => [
     { imgSrc: "images/Sasuke.jpeg", name: "Sasuke"},
 ];
 
-//Randomize
+// Randomize
 const randomize = () => {
     const cardData = getData();
     cardData.sort(() => Math.random() - 0.5);
```

Comment:
This does **not** randomize the array, it is strongly skewed. I very strongly suggest investigating and understanding why this is so; the reasoning is very interesting to explore. You can try running it yourself by hand to see what happens. Replace this with something that properly sorts the data (there are several options, you don't need to implement it yourself as its a little advanced)